samples/remote_attestation/README.md

```diff
@@ -43,7 +43,7 @@ There are two types of attestation:
 
 ### Secure Communication Channel
 
-Remote Attestation alone is not enough for the remote party to be able to securely deliver their secrets to the requesting enclave. Securely delivering services requires a secure communication channel. Using Asymmetric/Public-Key encryption is a mechanism for establishing such a channel.
+Remote Attestation alone is not enough for the remote party to be able to securely deliver their secrets to the requesting enclave. Securely delivering services requires a secure communication channel which is usually guaranteed by Transport Layer Security (TLS). Using Asymmetric/Public-Key encryption in conjunction with generating a symmetric key based on the sealing key provides an alternate mechanism for establishing such a channel in the absence of TLS support.
```

Comment:
Change "usually guaranteed" to "often guaranteed" (which makes no assertion that it's the majority).
Also change "channel in the absence of TLS support" to "non-TLS channel" since one can choose to not use TLS even if there is TLS support.

Reply:
ok, will incorporate.